tasks/extract_filepaths_for_granule/bin/run_tests.sh

```diff
@@ -97,11 +98,11 @@ let return_code=$?
 check_rc $return_code "ERROR: Potential security or code issues found."
 
 
-## Check code third party libraries for CVE issues
-echo "INFO: Running checks on third party libraries ..."
-safety check -r requirements.txt -r requirements-dev.txt
-let return_code=$?
-check_rc $return_code "ERROR: Potential security issues third party libraries."
+# ## Check code third party libraries for CVE issues
```

Comment:
Did you mean to comment these lines out?

Reply:
oops was testing a few things.

reverted